src/FlatSharp.Compiler/FlatSharpCompiler.cs

```diff
@@ -522,7 +526,7 @@ void OnDataReceived(object sender, DataReceivedEventArgs args)
                         Options = localOptions,
                         InputHash = inputHash,
                         PreviousAssembly = assembly,
-                        TypeModelContainer = TypeModelContainer.CreateDefault(),
+                        TypeModelContainer = TypeModelContainer.CreateDefault().WithUnitySupport(),
```

Comment:
Wondering if this should be a compiler switch like `--enable-unity-support`? I don't think I have strong feelings either way, so it's probably fine.

Reply:
I have added `--unity-assembly-path`, which does also require a path argument, to enable Unity support.